src/cascadia/WindowsTerminal/WindowEmperor.cpp

```diff
@@ -142,6 +142,9 @@ void WindowEmperor::WaitForWindows()
         TranslateMessage(&message);
         DispatchMessage(&message);
     }
+
+    _finalizeSessionPersistence();
+    TerminateProcess(GetCurrentProcess(), 0);

```

Comment:
this conflicts with #16575  right?

Reply:
Yep, this is just a reminder for me for when I resolve the merge conflicts.